frontend/src/pages/faq.page.tsx

```diff
@@ -229,6 +229,58 @@ const Faq: NextPage = () => {
                   <p />
                 </Localized>
               </QAndA>
+              <QAndA
+                id="faq-acceptable-use"
+                question={l10n.getString(
+                  "faq-question-acceptable-use-question"
+                )}
+              >
+                <Localized
+                  id="faq-question-acceptable-use-answer-a-html"
+                  vars={{
+                    url: "https://www.mozilla.org/about/legal/acceptable-use/",
+                    attrs: "",
+                  }}
+                  elems={{
+                    a: (
+                      <a href="https://www.mozilla.org/about/legal/acceptable-use/" />
+                    ),
+                  }}
+                >
+                  <p />
+                </Localized>
+                <ul>
+                  <li>
+                    {l10n.getString(
+                      "faq-question-acceptable-use-answer-measure-account"
+                    )}
+                  </li>
+                  <li>
+                    {l10n.getString(
+                      "faq-question-acceptable-use-answer-measure-unlimited-payment"
+                    )}
+                  </li>
+                  <li>
+                    {l10n.getString(
+                      "faq-question-acceptable-use-answer-measure-rate-limit"
+                    )}
+                  </li>
+                </ul>
+                <Localized
+                  id="faq-question-acceptable-use-answer-b-html"
+                  vars={{
+                    url: "https://www.mozilla.org/about/legal/terms/firefox-relay/",
+                    attrs: "",
+                  }}
+                  elems={{
+                    a: (
+                      <a href="https://www.mozilla.org/about/legal/terms/firefox-relay/" />
```

Comment:
suggestion: Can open all FAQ links in a new tab? 

From the Djagon frontend, we add the following attrs to each link in the FAQ: https://github.com/mozilla/fx-private-relay/blob/main/static/js/app.js#L362-L373

Note that the `noopener` may [no longer be necessary](https://github.com/whatwg/html/issues/4078).

Reply:
7b1a05b5

Note that the linter (configured to use the recommended default rules by ESLint and Next.js) added both parameters automatically on commit. Better safe than sorry, I suppose :)